detekt-config.yml

```diff
@@ -109,7 +109,7 @@ empty-blocks:
   EmptyForBlock:
     active: true
   EmptyFunctionBlock:
-    active: true
+    active: false
```

Comment:
Wait... Why do we need to disable this validation?

Reply:
@hoangnguyen92dn To allow empty function blocks (such as overriding) https://github.com/nimblehq/android-templates/blob/da9680602cf4bb08989c5ef97ac6f94c7ec6a4e5/app/src/main/java/co/nimblehq/ui/screens/MainActivity.kt#L15 🤓